biogrid/libs/biogrid-simulator/src/lib/biogrid/biogrid-class.ts

```diff
@@ -7,14 +7,22 @@ import {
   ItemPosition,
   Energy,
   Battery,
-  GridItem
+  GridItem,
+  Distance,
 } from '@biogrid/grid-simulator';
-import { LARGE_BATTERY, SMALL_BATTERY, SOLAR_PANEL, GRID_ITEM_NAMES, RESISTANCE } from '../config';
+import {
+  LARGE_BATTERY,
+  SMALL_BATTERY,
+  SOLAR_PANEL,
+  GRID_ITEM_NAMES,
+  RESISTANCE,
+  GRID_DISTANCES,
```

Comment:
here and above, if the imports use everything in the file, feel free to use `import *` instead